frontend/src/components/MainSearch/MainSearch.js

```diff
@@ -38,6 +40,10 @@ class MainSearch extends Component {
     }
   }
 
+  togglePanel(event) {
```

Comment:
If we're not going to use the event for anything, we don't really need it in the arguments.